src/convert-package.ts

```diff
@@ -12,11 +12,12 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+import * as fse from 'fs-extra';
 import * as path from 'path';
 import {Analysis, Analyzer, FSUrlLoader, InMemoryOverlayUrlLoader, PackageUrlResolver, ResolvedUrl} from 'polymer-analyzer';
 
 import {ConversionSettings, createDefaultConversionSettings, PartialConversionSettings} from './conversion-settings';
-import {generatePackageJson, readJson, writeJson} from './manifest-converter';
+import {generatePackageJson, writeJson} from './manifest-converter';
```

Comment:
Is there a benefit to maintaining the `writeJson` instead of using `fse.writeJson`?

Reply:
It converts `\n` to `os.EOL`. `fse.writeJson` actually supports this, but it's not in the latest typings. I might send a PR to add it to the typings. @FredKSchott Do we actually need to do this `\n` to `EOL` conversion for windows?

Reply:
@aomarks Without the `\n` to `EOL` conversions, it will cause the expected fixtures test to fail due to size differences.

Reply:
Gotcha, makes sense. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23831